client/src/models/Report.js

```diff
@@ -288,6 +288,16 @@ export default class Report extends Model {
       )
   })
 
+  static autocompleteQuery = "uuid, intent, author { uuid, name, rank, role }"
+
+  static autocompleteTemplate(report) {
```

Comment:
I think this is not being used anywhere.

Reply:
It is used to retrieve the report information needed for the auto-complete when searching ANET entities.

Reply:
As far as I can see it it is no longer used. It is only defined in all models, but not being used.

Reply:
Try to remove the line and serach for a report, you will see that it fails to fetch the data.

Reply:
The `autocompleteTemplate` should no longer be used anywhere.

Reply:
And `Report.autocompleteQuery` is also unused.

Reply:
autocompleteTemplate is really not being used, I removed it and I can still search for reports. I do get in the console the error: "/files/mara/workspace-ncia-anet/anet/client/src/models/Report.js
  10:8  error  'React' is defined but never used  no-unused-vars". So that one needs to be removed too, maybe that's why it didn't work for you.

Reply:
Indeed.